getusermedia.html

```diff
@@ -791,9 +791,9 @@ <h4>Life-cycle</h4>
         <p>When a {{MediaStreamTrackState/"live"}}, [= track/muted | unmuted =], and
         [= track/enabled =] track sourced by a device exposed
         by {{MediaDevices/getUserMedia()}} becomes either
-        [= track/muted or [= track/enabled | disabled =],
-        and this brings <em>all</em> tracks connected to the device (regardless
-        of browsing context) to be either
+        [= track/muted =] or [= track/enabled | disabled =],
+        and this brings <em>all</em> tracks connected to the device (across all
+        {{Document}}s the user agent operates) to be either
```

Comment:
Review feedback: Consider [transferable tracks to dedicated workers](https://w3c.github.io/mediacapture-extensions/#transferable-mediastreamtrack) (whose lifespan is still tied to a document, but it's perhaps better to say navigable, to avoid confusion.
```suggestion
        [=navigables=] the user agent operates) to be either
```